Mathlib/MeasureTheory/Measure/FiniteMeasure.lean

```diff
@@ -136,6 +137,20 @@ lemma coeFn_def (μ : FiniteMeasure Ω) : μ = fun s ↦ ((μ : Measure Ω) s).t
 lemma coeFn_mk (μ : Measure Ω) (hμ) :
     DFunLike.coe (F := FiniteMeasure Ω) ⟨μ, hμ⟩ = fun s ↦ (μ s).toNNReal := rfl
 
+/-- The type of probability measures is a measurable space when equipped with the Giry monad. -/
+instance {α : Type*} [MeasurableSpace α] : MeasurableSpace (FiniteMeasure α) :=
+    Subtype.instMeasurableSpace
+
+/-- The set of all finite measures is a measurable set in the Giry monad -/
+lemma finite_measures_measurable_set {α : Type*} [MeasurableSpace α] :
```

Comment:
Two remarks about names:
- the definition for measurable sets is `MeasurableSet`, hence we use `measurableSet` in the lemma names (and not `measurable_set`). Same for `IsFiniteMeasure`.
- we put the property first: `measurableSet_isFiniteMeasure` and not `isFiniteMeasure_measurableSet`
```suggestion
lemma measurableSet_isFiniteMeasure {α : Type*} [MeasurableSpace α] :
```